apps/ios/Shared/Views/Onboarding/WhatsNewView.swift

```diff
@@ -428,6 +428,37 @@ private let versionDescriptions: [VersionDescription] = [
             )
         ]
     ),
+    VersionDescription(
+        version: "v6.0",
+        post: URL(string: "https://simplex.chat/blog/20240814-simplex-chat-vision-funding-v6-private-routing-new-user-experience.html"),
+        features: [
+            FeatureDescription(
+                icon: "arrow.forward",
+                title: "Private message routing 🚀",
+                description: "Protects your IP address and connections."
+            ),
+            FeatureDescription(
+                icon: "person.text.rectangle",
+                title: "Your contacts your way",
```

Comment:
cheesy.. why can't it be simply "Better contact management"

Reply:
Because it's boring...